config/Coldbox.cfc

```diff
@@ -54,7 +54,7 @@ component{
 		// Module Directives
 		modules = {
 			//Turn to false in production, on for dev
-			autoReload = false
+			autoReload = true
```

Comment:
Why autoreload the shell? This just causes more dev issues than help in my experience?